docker-compose.yaml

```diff
@@ -10,24 +10,26 @@ services:
     - ./docker/env
     expose:
     - 5432
+    ports:
+    - 127.0.0.1:5432:5432
     networks:
       mobot:
         ipv4_address: 10.200.0.5
 
   signald:
-    image: mobilecoin/signald:0.12.0-mc.0.0.3-staging
+    image: mobilecoin/signald:v0.12.1
     volumes:
     - signald:/signald
     expose:
     - 15432
     ports:
-    - "15432:15432"
+    - 127.0.0.1:15432:15432
     networks:
       mobot:
         ipv4_address: 10.200.0.6
 
   full-service:
-    image: mobilecoin/full-service:v1.0.0-pre.5.testnet
+    image: mobilecoin/full-service:v1.0.1-testnet
```

Comment:
How is this differentiated for mainnet when we move to prod?

Reply:
Yes mainnet build of full service is `mobilecoin/full-service:v1.0.1`